avm/res/consumption/budget/README.md

```diff
@@ -267,7 +267,7 @@ module budget 'br/public:avm/res/consumption/budget:<version>' = {
 
 | Parameter | Type | Description |
 | :-- | :-- | :-- |
-| [`actionGroups`](#parameter-actiongroups) | array | List of action group resource IDs that will receive the alert. Required if neither `contactEmails` nor `contactEmails` was provided. |
+| [`actionGroups`](#parameter-actiongroups) | array | List of action group resource IDs that will receive the alert. Required if neither `contactEmails` nor `contactRoles` was provided. |
```

Comment:
Hey @the-blesss, good catch! Since the README.md is autogenerated, to update the description typo we should proceed in the following way:
Fix the typo in the corresponding main.bicep template @description
Run the Set-AVMmodule script as described [here](https://azure.github.io/Azure-Verified-Modules/contributing/bicep/bicep-contribution-flow/generate-bicep-module-files/) to generate both the compiled main.json and README.md documentation.

Please let us know if you have issues with the above and thanks for your contribution!